packages/manager/apps/pci-private-network/public/translations/listing/Messages_fr_FR.json

```diff
@@ -23,5 +23,8 @@
   "pci_projects_project_network_private_back_to_list": "Voir tous les réseaux privés",
   "pci_projects_project_network_private_subnet_create": "Créer un sous-réseau",
   "pci_projects_project_network_private_ip_version": "IP Version",
-  "pci_projects_project_network_private_action": "Actions"
+  "pci_projects_project_network_private_action": "Actions",
+  "pci_projects_project_network_private_subnet_show": "Voir les sous-réseaux",
+  "ACTIVE": "actif",
+  "DISABLED": "désactivé"
```

Comment:
These translations seems weird : one is an adjective, the other one is a past participle, and the key might not be interpreted correctly by translation team

Either you should add context about what is active or disabled, or use common translations